src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+// import { useState } from 'react';
```

Comment:
Kindly take away every commented piece of code from your project as it makes your codebase messy and gives it a less professional look😉.